deploy/500-controller.yaml

```diff
@@ -32,17 +32,3 @@ spec:
                   fieldPath: metadata.name
             - name: CONTROLLER_NAME
               value: "shipwright-build"
-          livenessProbe:
-            exec:
-              command:
-                - stat
-                - /tmp/shipwright-build-ready
-            initialDelaySeconds: 5
-            periodSeconds: 10
-          readinessProbe:
-            exec:
-              command:
-                - stat
-                - /tmp/shipwright-build-ready
-            initialDelaySeconds: 5
-            periodSeconds: 10
```

Comment:
Here is an alternative using the metrics endpoint:

```yaml
        ports:
        - containerPort: 8383
          name: metrics-port
        livenessProbe:
          httpGet:
            path: /metrics
            port: metrics-port
          initialDelaySeconds: 5
          periodSeconds: 10
        readinessProbe:
          httpGet:
            path: /metrics
            port: metrics-port
          initialDelaySeconds: 5
          periodSeconds: 10
```

We will likely need to use this downstream as we require a probe. It is probably as good as the Tekton solution.

Reply:
Do you require both a liveness probe and a readiness probe?

Reply:
I do not know, would need to check. But, you are right, we probably should here only define the liveness probe because there is no service defined. In downstream we have our own deployment yaml anyway, so, if we would have to specify a readiness probe, we can always deviate from what is defined in upstream.

Reply:
If you have your own downstream deployment anyway, I think I'd prefer to remove both probes here for now, until we decide there's real value in adding them. Does that sound okay to you?

Reply:
I think I am okay. The risk then is that we use something in downstream (a probe based on the metrics endpoint) that is not "tested" upstream. But, if it would stop working and not being repairable, then I think we could fairly quickly introduce a new metrics endpoint like Tekton does.

Reply:
We need both probes for some required standards/policies downstream. I will prefer to keep both in upstream so that we can continuously test them, if any vendor downstream does not require them, then they should remove it. PR looks good for me.